downstream/modules/eda/proc-eda-set-up-rulebook-activation.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc-eda-set-up-rulebook-activation"]
+
+= Setting up a rulebook activation
+
+[role="_abstract"]
+
+Rulebook activations are rulebooks that have been activated to run.
+
+== Prerequisites
+
+* You are logged in to the {EDAcontroller} Dashboard as a Content Consumer.
+* You have set up a project.
+* You have set up a decision environment.
+* You have set up an authentication token.
+
+== Procedure
+
+. Navigate to the {EDAcontroller} Dashboard.
+. From the navigation bar, select btn:[Rulebook Activations] → btn:[Create rulebook activation].
+. Insert the following: 
```

Comment:
Suggest changing to navigation panel and change btn to menu as follows:
select menu:Rulebook Activations[Create rulebook activation].